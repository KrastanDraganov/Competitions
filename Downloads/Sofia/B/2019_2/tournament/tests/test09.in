100
64 2 4 5 6 7 10 11 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 43 44 47 48 50 51 52 53 54 55 56 58 59 61 62 63 65 66 68 72 73 74 75 76 78 80 81 82 83 84 85 86 87 91 92 94 96 97 98
95 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 69 70 72 73 74 75 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99
58 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 75 76 78 80 81 82 84 86 87 91 92 94 96 97 98
60 0 2 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 97 98
20 5 7 18 22 24 25 26 31 39 43 44 53 58 59 68 74 75 84 87 97
5 21 25 39 79 85
51 4 5 7 10 13 15 16 17 18 20 21 22 24 25 26 29 31 32 35 43 44 47 51 52 53 54 55 56 58 59 61 62 65 66 68 70 72 73 74 75 76 78 80 81 82 84 85 87 92 94 97
13 5 18 25 26 39 43 44 59 68 75 84 87 97
65 0 3 4 5 6 7 10 11 13 15 16 17 20 21 22 24 25 26 29 30 31 32 34 35 36 39 43 44 47 48 50 51 52 53 54 55 56 58 59 61 62 63 66 68 72 73 74 75 76 78 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98
89 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 36 38 39 41 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
49 4 5 7 13 15 16 17 18 20 21 22 24 25 26 29 31 32 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
53 4 5 6 7 10 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 87 92 94 97 98
75 0 2 3 4 5 6 7 8 10 11 13 14 15 16 17 18 20 21 22 24 25 26 27 29 30 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 78 80 81 82 83 84 85 86 87 91 92 94 95 96 98 99
32 0 4 5 7 18 22 24 25 26 29 31 39 43 44 50 51 52 53 55 58 59 65 68 75 76 77 78 84 85 92 94 97
76 0 2 3 4 5 6 7 8 10 11 13 15 16 18 20 21 22 24 25 26 27 29 30 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 95 96 97 98 99
47 4 5 7 13 16 17 18 20 21 22 24 25 26 29 31 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
42 4 5 7 13 18 21 22 24 25 26 29 31 32 39 43 44 50 51 52 53 54 55 56 58 59 61 65 68 72 73 74 75 76 78 80 84 85 87 92 94 97 99
44 4 5 7 13 14 16 18 21 22 24 25 26 29 30 31 37 39 43 44 50 51 53 54 55 56 58 59 61 65 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
9 5 8 39 40 42 58 73 75 87
81 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 20 21 22 24 25 26 27 29 30 31 32 34 35 36 39 41 43 44 45 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 95 96 97 98 99
44 4 5 7 13 16 17 18 21 22 23 24 25 29 31 39 43 44 47 50 51 52 53 54 55 56 58 59 61 65 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
32 4 7 13 18 22 24 25 26 29 31 39 43 44 50 51 52 53 55 56 58 59 65 68 75 76 78 84 85 87 92 94 97
18 5 7 18 25 26 37 39 43 44 48 53 59 62 68 75 84 87 97
94 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 21 22 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 66 67 68 69 70 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 89 90 91 92 93 94 95 96 97 98 99
19 5 7 18 22 25 26 31 39 43 44 45 53 54 59 68 75 84 87 97
8 18 38 39 44 53 59 75 87
14 5 18 20 25 39 43 44 53 55 59 75 77 84 87
75 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 95 96 97 98 99
91 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 36 37 38 39 41 42 43 44 45 46 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
21 4 5 7 18 22 24 25 26 31 39 43 44 53 58 59 65 68 75 84 87 97
62 2 3 4 5 6 7 10 11 13 15 16 18 20 21 22 24 25 26 29 31 32 34 35 36 39 43 44 47 50 51 52 53 54 55 56 57 58 59 62 63 64 65 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 96 97
15 5 7 18 22 25 26 39 44 53 59 68 75 84 87 97
48 4 5 7 13 15 17 18 20 21 22 24 25 26 29 31 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97 98
79 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 34 35 36 39 41 42 43 44 45 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 76 78 79 80 81 82 83 86 90 91 92 94 95 96 98 99
53 4 5 6 7 10 13 15 16 17 18 20 21 22 24 25 26 29 31 32 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 72 73 74 75 76 78 80 81 82 84 85 87 92 94 97 98
60 2 3 4 5 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 96 97 98
53 4 5 6 7 10 13 15 16 17 18 20 21 22 24 25 26 29 31 32 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 69 72 73 74 75 76 78 80 81 82 84 85 87 92 94 97
87 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 24 25 26 27 29 30 31 32 33 34 35 36 38 39 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
84 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 26 27 29 30 31 32 33 34 35 36 39 41 43 44 45 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 92 93 94 95 96 97 98 99
3 6 27 79
90 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 19 20 21 22 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 41 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
71 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 37 39 43 44 46 47 48 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98
89 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 19 20 21 22 24 25 26 27 29 30 31 32 34 35 36 37 38 39 40 41 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 62 63 64 65 66 67 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 96 97 98 99
10 5 18 25 31 39 44 53 59 75 87
5 5 18 39 75 87
78 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 20 21 22 25 26 27 29 30 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 76 78 79 80 81 83 84 85 86 87 90 91 92 94 95 96 97 98 99
85 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 36 38 39 43 44 45 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 70 72 73 74 75 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
45 4 5 7 13 16 17 18 21 22 24 25 26 28 29 31 39 43 44 50 51 52 53 54 55 56 58 59 61 65 68 72 73 74 75 76 78 80 81 83 84 85 87 92 94 97
62 2 3 4 5 6 7 10 11 13 15 16 17 18 20 21 24 25 26 29 30 31 32 34 35 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 63 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 96 97 98
73 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 41 43 44 47 48 50 51 52 53 54 55 56 58 59 60 61 62 63 65 66 68 71 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98
30 4 5 6 7 18 22 24 25 26 29 31 39 43 44 51 52 53 55 58 59 68 75 76 78 84 85 87 92 94 97
30 4 5 7 18 22 24 25 26 29 31 39 43 44 52 53 55 58 59 68 75 76 78 79 84 85 87 89 92 94 97
26 4 5 7 17 18 22 24 25 26 29 31 39 43 44 53 58 59 68 75 76 78 84 87 92 94 97
10 5 7 18 39 44 59 75 76 80 87
36 4 5 7 13 18 21 22 25 26 29 31 39 43 44 50 51 52 53 55 58 59 65 68 72 73 74 75 76 78 80 84 85 87 92 94 97
25 4 5 7 18 22 24 25 29 31 39 43 44 52 53 58 59 68 75 76 78 84 87 92 94 97
39 4 5 7 11 13 18 22 24 25 26 29 31 39 43 44 50 51 52 53 54 55 58 59 61 65 68 72 73 74 75 76 78 80 84 85 87 92 94 97
81 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 31 32 33 34 35 36 39 41 43 44 45 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 64 65 66 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 95 96 97 98 99
20 5 7 11 22 24 25 26 31 39 43 44 53 59 64 68 75 79 84 87 97
7 5 18 39 44 62 75 87
71 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 41 43 44 47 48 50 51 52 53 54 55 56 58 59 61 62 63 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98
40 4 5 7 13 18 21 22 24 25 26 29 30 31 39 42 43 44 50 51 52 53 54 55 58 59 65 68 72 73 74 75 76 78 80 84 85 87 92 94 97
43 4 5 7 13 16 17 18 20 21 24 25 26 29 31 39 43 44 47 50 51 52 53 54 55 56 58 61 65 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
64 2 3 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 35 36 39 41 43 44 47 50 51 52 53 54 55 56 57 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 96 97 98
72 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 59 60 61 62 63 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98 99
31 4 5 7 8 18 22 23 24 25 26 31 39 43 44 48 50 51 52 53 55 58 59 68 76 78 84 85 87 92 94 97
47 4 5 7 13 16 17 18 20 21 22 24 25 26 29 31 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 68 70 72 73 74 75 76 78 80 81 84 85 87 92 94 97
85 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 36 39 40 41 43 44 45 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 95 96 97 98 99
15 1 5 18 25 26 34 39 43 44 53 59 75 84 87 97
87 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 38 39 41 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
76 0 2 3 4 5 7 8 10 11 13 14 15 16 17 18 20 21 22 24 25 26 27 29 30 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 95 96 97 98 99
97 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 98 99
34 4 5 7 13 18 21 22 24 25 26 29 31 39 43 44 50 51 52 53 55 58 59 65 68 75 76 78 80 84 85 87 92 94 97
35 4 5 7 13 21 22 24 25 26 29 31 39 43 44 50 51 52 53 55 58 59 65 68 72 75 76 78 80 84 85 87 89 92 94 97
36 2 5 7 13 18 21 22 24 25 26 29 31 39 43 44 50 51 52 53 55 58 59 65 68 72 73 75 76 78 80 84 85 87 92 94 97
3 5 39 65
26 1 4 5 7 12 18 22 24 25 26 29 31 39 43 44 46 58 59 68 75 78 84 87 92 94 97
93 0 2 3 4 5 6 7 8 9 10 11 12 14 15 16 17 18 19 20 21 22 24 25 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 89 90 91 92 93 94 95 96 97 98 99
24 4 5 7 18 22 24 25 26 29 31 39 43 44 53 58 59 68 75 83 84 87 92 94 97
66 0 2 3 4 6 7 8 10 11 12 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 43 44 47 48 50 52 53 54 55 56 59 61 62 63 65 66 68 72 73 74 75 76 78 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98
32 4 5 7 13 18 21 22 24 25 26 29 31 39 43 44 50 51 52 55 58 59 65 68 75 76 78 84 85 87 92 94 97
41 4 5 7 13 16 18 21 22 24 25 26 29 31 39 43 44 50 51 52 53 54 55 56 58 59 61 65 68 72 73 74 75 76 78 80 84 85 87 92 94 97
51 4 5 7 10 13 15 16 17 18 20 21 22 24 25 26 29 31 32 39 43 44 45 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 84 85 87 92 94 97
62 2 3 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 43 44 48 50 51 52 53 54 55 56 58 59 61 62 63 65 66 68 72 73 74 75 76 80 81 82 84 85 86 91 92 94 96 97 98
11 5 18 25 33 39 43 44 53 59 75 87
28 2 4 7 18 22 24 25 26 29 31 33 39 43 44 52 53 55 58 59 68 75 76 78 84 87 92 94 97
57 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 87 91 92 94 97 98
6 5 13 33 39 75 83
97 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 89 90 91 92 93 94 95 96 97 98 99
92 0 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 72 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 93 94 95 96 97 98 99
68 0 2 3 4 5 6 7 10 11 12 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 43 44 47 48 50 51 52 53 54 55 56 58 59 61 62 63 65 66 68 72 73 74 75 76 78 80 81 82 83 84 85 86 87 91 92 94 96 97 98
57 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 38 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 87 92 93 97 98
22 4 5 7 18 22 24 25 26 29 31 39 43 44 53 58 59 68 75 84 87 94 97
83 0 2 3 4 5 6 7 8 10 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 29 30 31 32 33 34 35 36 39 41 43 44 45 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 68 70 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 92 94 95 96 97 98 99
23 4 5 7 18 22 24 25 26 29 31 39 43 44 53 58 59 67 68 75 84 87 91 97
76 0 2 3 4 5 6 7 8 10 11 13 15 16 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 41 42 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97 98 99
59 3 4 5 6 7 10 11 13 15 16 17 18 20 21 22 24 25 26 29 31 32 34 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 86 87 91 92 94 97 98
15 5 12 18 25 26 33 39 43 44 53 59 71 75 84 87
54 4 5 6 7 10 13 15 16 17 18 20 21 22 24 25 26 29 30 31 36 39 43 44 47 50 51 52 53 54 55 56 58 59 61 62 65 66 68 72 73 74 75 76 78 80 81 82 84 85 87 92 94 97 99
71 0 2 3 4 5 6 7 8 10 11 13 15 17 18 20 21 22 24 25 26 29 30 31 32 34 35 36 39 41 43 44 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 65 66 68 72 73 74 75 76 78 79 80 81 82 83 84 85 86 87 90 91 92 94 96 97
