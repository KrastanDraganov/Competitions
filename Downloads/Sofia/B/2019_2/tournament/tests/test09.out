98 99 97 71 88 77 89 69 95 96 94 67 93 70 64 90 83 63 86 91 82 66 62 81 61 42 74 73 72 85 76 78 92 68 1 75 65 23 40 57 49 60 48 47 28 46 45 41 37 9 33 19 12 27 30 35 34 36 32 56 54 50 51 55 52 29 58 43 31 26 20 17 14 24 53 59 44 18 8 3 15 16 13 0 2 11 6 10 4 7 5 21 22 25 38 39 79 80 84 87 
